main.py

```
@@ -1,44 +1,437 @@
+    return "⚡ Bot 7/24 Aktivdir!"
+
+def run_flask():
+    app.run(host='0.0.0.0', port=8080)
+
+def keep_alive():
+    t = Thread(target=run_flask)
+    t.daemon = True
+    t.start()
+
+# --- AYARLAR --- BURANI DƏYİŞ
+TOKEN = "8756196909:AAHMJ53ExNhnm7ewDR_Gho01C6_g9R4aPL8" # TƏCİLİ DƏYİŞ @BotFather-dan
```

Ruff (0.15.12)

[error] Possible hardcoded password assigned to: "TOKEN"

(S105)